internal/common/events/events.go

```diff
@@ -375,3 +375,633 @@ func (e *HostRegistrationFailedSettingPropertiesEvent) FormatMessage() string {
     return e.format(&s)
 }
 
+//
```

Comment:
I believe this file is generated. GitHub doesn't recognize it that way. Please take a look how it could be marked as a generated file so GitHub would collapse it accordingly. 
This is not a blocker.